src/compose/types/page-block/file.ts

```diff
@@ -7,6 +7,13 @@ interface Options {
   mode: string;
   attachments: string[];
   hideFileName: boolean;
+  height: string;
```

Comment:
I think some of these properties can be numbers, like height, width, maxHeight, maxWidth, borderRadius, margin